components/doc/server.js

```diff
@@ -5,8 +5,16 @@ import { ServerComponent } from "../server/index.js";
 export class Doc extends ServerComponent {
   /**
    * @param {import("@fred").Context<import("@rari").DocPage>} context
-   */ render(context) {
+   */
+  render(context) {
     context.pageTitle = context.doc.pageTitle;
     return PageLayout.render(context, ReferenceLayout.render(context));
   }
+
+  /**
+   * @param {import("@fred").Context<import("@rari").DocPage>} context
+   */
```

Comment:
nit: explain what this method omits (even if it's comparably trivial from looking at render/renderSimple side-by-side